README.md

```diff
@@ -3,7 +3,7 @@
 
 ## FAQ
 1. dafuq?
-  Sometimes you need to be sarcastic on teh internets.
```

Comment:
This was intentional.
https://www.urbandictionary.com/define.php?term=Teh%20Interwebz
Perhaps it should be "teh interwebz"

Reply:
Well TIL "teh interwebz" :smile: